haskell/src/SwiftNav/SBP/Bootload.hs

```diff
@@ -68,6 +72,8 @@ instance Binary MsgBootloaderHandshakeResp where
     putWord32le msgBootloaderHandshakeRespFlags
     putByteString msgBootloaderHandshakeRespVersion
 
+$(deriveJSON defaultOptions ''MsgBootloaderHandshakeResp)
```

Comment:
out of curiosity, how does this formulate the field names - is it compatible with what we have in python? is it configurable?